core/storage/suggestion/gae_models.py

```diff
@@ -626,16 +662,18 @@ class CommunityContributionStatsModel(base_models.BaseModel):
     # suggestions are offered in and the values correspond to the total number
     # of reviewers who have permission to review translation suggestions in
     # that language.
-    translation_reviewer_counts_by_lang_code = ndb.JsonProperty(required=True)
+    translation_reviewer_counts_by_lang_code = ndb.JsonProperty(
```

Comment:
Why do you need the "default" arg, here and below? Shouldn't these fields be explicitly initialized when the model is created?

Reply:
This was brought up by @DubeySandeep in this [comment](https://github.com/oppia/oppia/pull/10708#discussion_r493193561), but I'll take the defaults out.